test/integration/transaction.test.js

```diff
@@ -142,6 +142,16 @@ describe(Support.getTestDialectTeaser('Transaction'), function() {
     ).to.eventually.be.rejected;
   });
 
+  it('does not allow queries immediatly after rollback call', function() {
+    var self = this;
+    return expect(
+      this.sequelize.transaction().then(function(t) {
+        t.rollback();
+        return self.sequelize.query('SELECT 1+1', {transaction: t, raw: true});
+      })
+    ).to.eventually.be.rejected;
```

Comment:
`to.eventually.be.rejectedWith(errorMessage)`, best to be specific about what kind of error we're expecting.